### PR TITLE
Avatar badge

### PR DESCRIPTION
**Pull request by hiteshverma27**
Deployed preview for avatar and badge. [Click here for preview](https://orca-deploy-one.netlify.app/)

**This pull request also contains landing page and basic site layout.**

### CODE REVIEW

**Review by Creator-singh — APPROVED** [review]
LGTM

File: Styles/avatar.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.avatar{
+    border-radius: 999px;
```
Comment (surbhi84) [inline Styles/avatar.css:2]: suggestion: percentage can be used instead of px.

**Review by surbhi84 — APPROVED** [review]
rest looks good to me.

File: Styles/avatar.css (hunk 1 above)
Comment (hiteshverma27) [reply Styles/avatar.css:2]: I took it from the figma design file but sure I will change that.

**Comment by rahul31agrawal** [comment]
Good job Hitesh, your CL looks awesome.

**Review by Bibin-Antony — APPROVED** [review]
Everything seems fine, could proceed.

**Pull request merged**